src/Imath/ImathNamespace.h

```diff
@@ -3,49 +3,51 @@
 // Copyright Contributors to the OpenEXR Project.
 //
 
+///
+/// @file  ImathNamespace.h
+///
+/// @brief The Imath library namespace
+///
+/// The purpose of this file is to make it possible to specify an
+/// IMATH_INTERNAL_NAMESPACE as a preprocessor definition and have all of the
+/// Imath symbols defined within that namespace rather than the standard
+/// Imath namespace.  Those symbols are made available to client code through
+/// the IMATH_NAMESPACE in addition to the IMATH_INTERNAL_NAMESPACE.
+///
+/// To ensure source code compatibility, the IMATH_NAMESPACE defaults to Imath
```

Comment:
I think these are fine for documentation in the header itself for human readers, but none of the `@file` stuff will currently show up in your readthedocs.

Another strategy might be to put these introductory materials right into the .rst itself.

Reply:
Converted to non-doxygen style.

Reply:
It was fine as it was, it doesn't hurt anything to be doxygen style. I just wanted to be sure you knew that those weren't ending up anywhere in the final docs.